components/gitpod-protocol/src/protocol.ts

```diff
@@ -972,6 +972,13 @@ export interface WorkspaceConfig {
     _featureFlags?: NamedWorkspaceFeatureFlag[];
 }
 
+// WorkspaceConfigContext is the context to help parse/generate workspace config
+export interface WorkspaceConfigContext {
```

Comment:
`WorkspaceConfigOptions` ?

why is it on protocol? Should not be in config-provider?

Reply:
new interface removed here https://github.com/gitpod-io/gitpod/pull/18723/commits/2756cafbad251e5522572d2a8e0f1b3e433fdef5